vm/src/stdlib/io.rs

```diff
@@ -357,7 +407,7 @@ pub fn io_open(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
         optional = [(mode, Some(vm.ctx.str_type()))]
     );
     // mode is optional: 'rt' is the default mode (open from reading text)
-    let mode_string = mode.map_or("rt".to_string(), |m| objstr::get_value(m));
+    let mode_string = mode.map_or("rt".to_string(), |m| get_value(m));
```

Comment:
I think `get_value` is too ambiguous name to be used without namespace here. `objstr::get_value` is easier to read. And it's against Rust guidelines. 

Also, could we use just `objstr::get_value` here, without creating new closure?

Reply:
This could probably just be a `match` in order to avoid unnecessary allocation.